app/controllers/myott/subscriptions_controller.rb

```diff
@@ -2,10 +2,45 @@ module Myott
   class SubscriptionsController < ApplicationController
     before_action :disable_search_form,
                   :disable_switch_service_banner,
-                  :disable_last_updated_footnote
+                  :disable_last_updated_footnote,
+                  :sections_chapters
```

Comment:
I think you could avoid needing a before_action by calling the method instead of the instance variable